src/lib/themes-test.js

```diff
@@ -33,6 +33,137 @@ describe("lib/themes", () => {
         expect(output).to.deep.equal(theme);
       });
     });
+    it("should migrate themes without popup/popup_text", () => {
+      const theme = {
+        images: {
+          headerURL: "./bg-000.svg",
+        },
+        colors: {
+          frame: { r: 12, g: 34, b: 56 },
+          toolbar: { r: 1, g: 2, b: 3, a: 1 },
+          toolbar_text: { r: 7, g: 8, b: 9 },
+        },
+        title: ["old theme without popup"],
+      };
+      const expectedTheme = {
+        images: { additional_backgrounds: ["./bg-000.svg"] },
+        colors: {
+          ...presetThemesContext("./default.json").colors,
+          frame: { r: 12, g: 34, b: 56 },
+          popup: { r: 12, g: 34, b: 56 }, // default to frame.
+          popup_text: { r: 7, g: 8, b: 9 }, // default to toolbar_text.
+          toolbar: { r: 1, g: 2, b: 3, a: 1 },
+          toolbar_text: { r: 7, g: 8, b: 9 },
+        },
+        title: ["old theme without popup"],
+      };
+      const output = subject(theme);
+      expect(output).to.deep.equal(expectedTheme);
+    });
+    it("should migrate deprecated theme properties", () => {
+      const deprecatedTheme = {
+        images: {
+          headerURL: ["dummy.png"],
```

Comment:
headerURL is meant to be a string

also, now I see that only background images that are included in bgImages are meant to be still part of the normalized theme, would you mind to add an inline comment on top of this line to mention that it is expected that dummy.png will not be part of `images.additional_backgrounds` in the normalized theme?